scripts/ampHtmlValidator/validator/index.js

```diff
@@ -71,14 +71,27 @@ const runValidator = async () => {
   return Promise.all(
     urlsToValidate.map(url => validate({ validator, url })),
   ).then(results => {
-    results.forEach(result => {
+    // Temporary fix for 'INVALID_URL_PROTOCOL' errors
+    const filteredResults = results.map(result => {
+      if (result.status !== 'PASS') {
+        const isProtocolFail = result.errors.every(
+          error => error.code === 'INVALID_URL_PROTOCOL',
+        );
+        if (isProtocolFail) {
+          // eslint-disable-next-line no-param-reassign
+          result = { ...result, status: 'PASS' };
```

Comment:
Would this work? So that we don't need to reassign the parameter? 

```suggestion
          result.status = 'PASS';
```

Reply:
It is working